CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.10.0
+
+* Enhancements
+  * Combined individual port executables into a single multi-call port executable.
+  * Improved start up performance refactoring uevent discovery and moving it to
+    the C port executable.
+  * Improved kernel logger performance by preventing a message copy through an
+    intermediate buffer.
+  * Turn off ENOBUFS messages.
```

Comment:
Slightly more context:

  * Instruct the Linux kernel to ignore `ENOBUFS` errors when `uevents` are generated faster than they can be handled. This chooses ignoring events over getting into a potential restart loop if events get generated too fast on device enumeration. It has only been seen on a fast x86 machine with many devices.